lib/spandex_tesla.ex

```diff
@@ -30,7 +30,7 @@ defmodule SpandexTesla do
   # in application.ex
   :telemetry.attach(
     "spandex-tesla-tracer",
-    [:tesla, :request],
+    [:tesla, :request, :start],
     &SpandexTesla.handle_event/4,
     nil
   )
```

Comment:
`SpandexTesla.Telemetry.attach()` 

Maybe here we should also use the new introduce command WDYT?